Chary/Assignments/Functions_part_2/8_occurances_vowels.py

```diff
@@ -0,0 +1,29 @@
+#Write a program count number of occurrences of vowels in a given string
+
+def occurrences_of_vowels(str2):
+    vowels="AEIOU"
+    count_vowels={}
+    for ch2 in str2:
+        if ch2 in vowels and ch2 not in count_vowels :
```

Comment:
improve the code readability in this case, you can use nested if statements

Reply:
fixed.